src/tests/team.rs

```diff
@@ -54,13 +52,12 @@ fn weird_name() {
             "{:?}", json.errors);
 }
 
-// Test adding team without second `:`
+// // Test adding team without second `:`
```

Comment:
Nit: Is this line so nice we need to comment it out twice? ;) This happens a few times in this file/commit.